numpy/core/tests/test_multiarray.py

```diff
@@ -346,17 +346,9 @@ def test_int_subclassing(self):
 
         numpy_int = np.int_(0)
 
-        if sys.version_info[0] >= 3:
-            # On Py3k int_ should not inherit from int, because it's not
-            # fixed-width anymore
-            assert_equal(isinstance(numpy_int, int), False)
-        else:
-            # Otherwise, it should inherit from int...
-            assert_equal(isinstance(numpy_int, int), True)
-
-            # ... and fast-path checks on C-API level should also work
-            from numpy.core._multiarray_tests import test_int_subclass
-            assert_equal(test_int_subclass(numpy_int), True)
+        # On Py3k int_ should not inherit from int, because it's not
```

Comment:
```suggestion
        # int_ does inherit from Python int, because it is fixed width
```
Just a thought. (removing next line)

Reply:
I'm confused by your comment, it says the opposite of the old comment and seems to not agree with the assert.
did you mean 'int_ does**n't** inherit from Python int? or does this test need to be changed?

Reply:
Whoops, forgot to add the *not*, yes.